### PR TITLE
🖍 CSS / Styling: Adjust the layout when choose "is one of" in "select operation"

### PR DESCRIPTION
**Pull request by qixiaojian310**
Change the layout and controller of the multi-selection

1. When input many contents:
ddfd2a93fbd4ea7b2b12a0e08fbf27a
which can be scrolled when it overflows.
2. When the width changes, it can extend.
图片
When width change：
图片

### CODE REVIEW

File: modules/front-end/src/app/core/components/find-rule/rule/rule.component.less [hunk 1]
```diff
@@ -20,14 +20,25 @@ section.rule-item {
     }
   }
 
+  .flexSelect {
+    position: relative;
```
Comment (cosmos-explorer) [inline modules/front-end/src/app/core/components/find-rule/rule/rule.component.less]: you can remove this line

File: modules/front-end/src/app/core/components/find-rule/rule/rule.component.less [hunk 2]
```diff
@@ -20,14 +20,25 @@ section.rule-item {
...
+    flex: 1;
+    max-width: calc(100% - 500px);
+    nz-select {
+      position: relative;
```
Comment (cosmos-explorer) [inline modules/front-end/src/app/core/components/find-rule/rule/rule.component.less]: this line can be removed

File: modules/front-end/src/styles/nz-select.less [hunk 3]
```diff
@@ -66,12 +68,24 @@ nz-select:not(.ant-select-customize-input) {
     }
 
     &.nz-select-36 .ant-select-selector {
-      height: 36px;
+      min-height: 36px;
+      max-height: 100px;
+      padding-top: 0;
+      padding-bottom: 0;
+      overflow-x: scroll;
```
Comment (cosmos-explorer) [inline modules/front-end/src/styles/nz-select.less]: This line add a horizontal scroll bar permannently, should remove this line

File: modules/front-end/src/app/core/components/find-rule/rule/rule.component.less [hunk 4]
```diff
@@ -20,14 +20,25 @@ section.rule-item {
...
+      max-width: 100%;
+      width: 100%;
+    }
+  }
+
   .value {
     nz-input-number {
       width: 100%;
-      line-height: 36px;
+      line-height: 34.4px;
```
Comment (cosmos-explorer) [inline modules/front-end/src/app/core/components/find-rule/rule/rule.component.less:35]: Does this really change anything? I think you can change it back

File: modules/front-end/src/app/core/components/find-rule/rule/rule.component.less [hunk 5]
```diff
@@ -20,14 +20,25 @@ section.rule-item {
...
     }
 
     nz-select {
-      line-height: 36px;
+      line-height: 34.4px;
```
Comment (cosmos-explorer) [inline modules/front-end/src/app/core/components/find-rule/rule/rule.component.less:39]: the same

File: modules/front-end/src/styles/nz-select.less (hunk 3 above)
Comment (qixiaojian310) [reply modules/front-end/src/styles/nz-select.less]: Have fixed

**Review by cosmos-explorer — APPROVED** [review]
LGTM

**Comment by cosmos-explorer** [comment]
fixed #550

**Pull request merged**